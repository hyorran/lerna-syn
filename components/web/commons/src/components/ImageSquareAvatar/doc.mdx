---
name: ImageSquareAvatar
menu: Commons
---

import { Playground, PropsTable } from 'docz'
import ImageSquareAvatar, { ComponentWithProps } from './'

# ImageSquareAvatar

A Simple ImageSquareAvatar Example 

## Basic Usage
<Playground>
  <div>
    <ImageSquareAvatar />
  </div>
</Playground>

## Props

<PropsTable of={ ComponentWithProps } />
