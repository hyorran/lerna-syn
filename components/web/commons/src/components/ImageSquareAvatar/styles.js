import Colors from '@syntesis/c-styles/src/styles/Colors'

export default () => ({
  avatar: {
    margin: 0,
    color: Colors.white,
    width: 40,
    fontSize: 12,
    borderRadius: 0,
    fontWeight: 'bold',
    '& img': {
      height: 'auto'
    }
  }
})

