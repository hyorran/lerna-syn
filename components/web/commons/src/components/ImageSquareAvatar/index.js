import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import Tooltip from '@material-ui/core/Tooltip';
import Avatar from '@material-ui/core/Avatar';

import styles from './styles'

function ImageSquareAvatar(props) {
  const {
    classes,
    value,
    src,
    tooltip
  } = props;

  return (
    <Grid container justify="center" alignItems="center">
      <Tooltip
        title={ `${ tooltip !== null && tooltip !== '' ? tooltip : value }` }
        placement="right"
      >
        <Avatar className={ classes.avatar } src={ src } />
      </Tooltip>
    </Grid>
  );
}

ImageSquareAvatar.propTypes = {
  /** Provided by material-ui. */
  classes: PropTypes.object.isRequired,
  /** - */
  src: PropTypes.string,
  /** - */
  tooltip: PropTypes.string,
  /** - */
  value: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number
  ]),
}

ImageSquareAvatar.defaultProps = {
  src: '',
  tooltip: '',
  value: ''
}

export default withStyles(styles)(ImageSquareAvatar)
export { ImageSquareAvatar as ComponentWithProps }
