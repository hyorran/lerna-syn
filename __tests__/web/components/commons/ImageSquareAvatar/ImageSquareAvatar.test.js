import React from 'react'
import ImageSquareAvatar from '@syntesis/c-commons/src/components/ImageSquareAvatar'
import { mount } from 'enzyme'

describe('<ImageSquareAvatar />', () => {
  it('rendered', () => {
    const wrapper = mount((
      <ImageSquareAvatar />
    ))
    expect(wrapper).toMatchSnapshot()
  })
})
