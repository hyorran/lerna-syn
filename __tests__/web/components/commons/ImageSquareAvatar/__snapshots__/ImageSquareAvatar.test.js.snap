// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImageSquareAvatar /> rendered 1`] = `
<WithStyles(ImageSquareAvatar)>
  <ImageSquareAvatar
    classes={
      Object {
        "avatar": "ImageSquareAvatar-avatar-1",
      }
    }
    src=""
    tooltip=""
    value=""
  >
    <WithStyles(Grid)
      alignItems="center"
      container={true}
      justify="center"
    >
      <Grid
        alignContent="stretch"
        alignItems="center"
        classes={
          Object {
            "align-content-xs-center": "MuiGrid-align-content-xs-center-14",
            "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-16",
            "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-15",
            "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-18",
            "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-17",
            "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-13",
            "align-items-xs-center": "MuiGrid-align-items-xs-center-10",
            "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-12",
            "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-11",
            "container": "MuiGrid-container-2",
            "direction-xs-column": "MuiGrid-direction-xs-column-5",
            "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-6",
            "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-7",
            "grid-lg-1": "MuiGrid-grid-lg-1-73",
            "grid-lg-10": "MuiGrid-grid-lg-10-82",
            "grid-lg-11": "MuiGrid-grid-lg-11-83",
            "grid-lg-12": "MuiGrid-grid-lg-12-84",
            "grid-lg-2": "MuiGrid-grid-lg-2-74",
            "grid-lg-3": "MuiGrid-grid-lg-3-75",
            "grid-lg-4": "MuiGrid-grid-lg-4-76",
            "grid-lg-5": "MuiGrid-grid-lg-5-77",
            "grid-lg-6": "MuiGrid-grid-lg-6-78",
            "grid-lg-7": "MuiGrid-grid-lg-7-79",
            "grid-lg-8": "MuiGrid-grid-lg-8-80",
            "grid-lg-9": "MuiGrid-grid-lg-9-81",
            "grid-lg-auto": "MuiGrid-grid-lg-auto-71",
            "grid-lg-true": "MuiGrid-grid-lg-true-72",
            "grid-md-1": "MuiGrid-grid-md-1-59",
            "grid-md-10": "MuiGrid-grid-md-10-68",
            "grid-md-11": "MuiGrid-grid-md-11-69",
            "grid-md-12": "MuiGrid-grid-md-12-70",
            "grid-md-2": "MuiGrid-grid-md-2-60",
            "grid-md-3": "MuiGrid-grid-md-3-61",
            "grid-md-4": "MuiGrid-grid-md-4-62",
            "grid-md-5": "MuiGrid-grid-md-5-63",
            "grid-md-6": "MuiGrid-grid-md-6-64",
            "grid-md-7": "MuiGrid-grid-md-7-65",
            "grid-md-8": "MuiGrid-grid-md-8-66",
            "grid-md-9": "MuiGrid-grid-md-9-67",
            "grid-md-auto": "MuiGrid-grid-md-auto-57",
            "grid-md-true": "MuiGrid-grid-md-true-58",
            "grid-sm-1": "MuiGrid-grid-sm-1-45",
            "grid-sm-10": "MuiGrid-grid-sm-10-54",
            "grid-sm-11": "MuiGrid-grid-sm-11-55",
            "grid-sm-12": "MuiGrid-grid-sm-12-56",
            "grid-sm-2": "MuiGrid-grid-sm-2-46",
            "grid-sm-3": "MuiGrid-grid-sm-3-47",
            "grid-sm-4": "MuiGrid-grid-sm-4-48",
            "grid-sm-5": "MuiGrid-grid-sm-5-49",
            "grid-sm-6": "MuiGrid-grid-sm-6-50",
            "grid-sm-7": "MuiGrid-grid-sm-7-51",
            "grid-sm-8": "MuiGrid-grid-sm-8-52",
            "grid-sm-9": "MuiGrid-grid-sm-9-53",
            "grid-sm-auto": "MuiGrid-grid-sm-auto-43",
            "grid-sm-true": "MuiGrid-grid-sm-true-44",
            "grid-xl-1": "MuiGrid-grid-xl-1-87",
            "grid-xl-10": "MuiGrid-grid-xl-10-96",
            "grid-xl-11": "MuiGrid-grid-xl-11-97",
            "grid-xl-12": "MuiGrid-grid-xl-12-98",
            "grid-xl-2": "MuiGrid-grid-xl-2-88",
            "grid-xl-3": "MuiGrid-grid-xl-3-89",
            "grid-xl-4": "MuiGrid-grid-xl-4-90",
            "grid-xl-5": "MuiGrid-grid-xl-5-91",
            "grid-xl-6": "MuiGrid-grid-xl-6-92",
            "grid-xl-7": "MuiGrid-grid-xl-7-93",
            "grid-xl-8": "MuiGrid-grid-xl-8-94",
            "grid-xl-9": "MuiGrid-grid-xl-9-95",
            "grid-xl-auto": "MuiGrid-grid-xl-auto-85",
            "grid-xl-true": "MuiGrid-grid-xl-true-86",
            "grid-xs-1": "MuiGrid-grid-xs-1-31",
            "grid-xs-10": "MuiGrid-grid-xs-10-40",
            "grid-xs-11": "MuiGrid-grid-xs-11-41",
            "grid-xs-12": "MuiGrid-grid-xs-12-42",
            "grid-xs-2": "MuiGrid-grid-xs-2-32",
            "grid-xs-3": "MuiGrid-grid-xs-3-33",
            "grid-xs-4": "MuiGrid-grid-xs-4-34",
            "grid-xs-5": "MuiGrid-grid-xs-5-35",
            "grid-xs-6": "MuiGrid-grid-xs-6-36",
            "grid-xs-7": "MuiGrid-grid-xs-7-37",
            "grid-xs-8": "MuiGrid-grid-xs-8-38",
            "grid-xs-9": "MuiGrid-grid-xs-9-39",
            "grid-xs-auto": "MuiGrid-grid-xs-auto-29",
            "grid-xs-true": "MuiGrid-grid-xs-true-30",
            "item": "MuiGrid-item-3",
            "justify-xs-center": "MuiGrid-justify-xs-center-19",
            "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-20",
            "justify-xs-space-around": "MuiGrid-justify-xs-space-around-22",
            "justify-xs-space-between": "MuiGrid-justify-xs-space-between-21",
            "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-23",
            "spacing-xs-16": "MuiGrid-spacing-xs-16-25",
            "spacing-xs-24": "MuiGrid-spacing-xs-24-26",
            "spacing-xs-32": "MuiGrid-spacing-xs-32-27",
            "spacing-xs-40": "MuiGrid-spacing-xs-40-28",
            "spacing-xs-8": "MuiGrid-spacing-xs-8-24",
            "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-8",
            "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-9",
            "zeroMinWidth": "MuiGrid-zeroMinWidth-4",
          }
        }
        component="div"
        container={true}
        direction="row"
        item={false}
        justify="center"
        lg={false}
        md={false}
        sm={false}
        spacing={0}
        wrap="wrap"
        xl={false}
        xs={false}
        zeroMinWidth={false}
      >
        <div
          className="MuiGrid-container-2 MuiGrid-align-items-xs-center-10 MuiGrid-justify-xs-center-19"
        >
          <WithStyles(Tooltip)
            placement="right"
            title=""
          >
            <Tooltip
              TransitionComponent={[Function]}
              classes={
                Object {
                  "popper": "MuiTooltip-popper-99",
                  "popperInteractive": "MuiTooltip-popperInteractive-100",
                  "tooltip": "MuiTooltip-tooltip-101",
                  "tooltipPlacementBottom": "MuiTooltip-tooltipPlacementBottom-106",
                  "tooltipPlacementLeft": "MuiTooltip-tooltipPlacementLeft-103",
                  "tooltipPlacementRight": "MuiTooltip-tooltipPlacementRight-104",
                  "tooltipPlacementTop": "MuiTooltip-tooltipPlacementTop-105",
                  "touch": "MuiTooltip-touch-102",
                }
              }
              disableFocusListener={false}
              disableHoverListener={false}
              disableTouchListener={false}
              enterDelay={0}
              enterTouchDelay={1000}
              interactive={false}
              leaveDelay={0}
              leaveTouchDelay={1500}
              placement="right"
              theme={
                Object {
                  "breakpoints": Object {
                    "between": [Function],
                    "down": [Function],
                    "keys": Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ],
                    "only": [Function],
                    "up": [Function],
                    "values": Object {
                      "lg": 1280,
                      "md": 960,
                      "sm": 600,
                      "xl": 1920,
                      "xs": 0,
                    },
                    "width": [Function],
                  },
                  "direction": "ltr",
                  "mixins": Object {
                    "gutters": [Function],
                    "toolbar": Object {
                      "@media (min-width:0px) and (orientation: landscape)": Object {
                        "minHeight": 48,
                      },
                      "@media (min-width:600px)": Object {
                        "minHeight": 64,
                      },
                      "minHeight": 56,
                    },
                  },
                  "overrides": Object {},
                  "palette": Object {
                    "action": Object {
                      "active": "rgba(0, 0, 0, 0.54)",
                      "disabled": "rgba(0, 0, 0, 0.26)",
                      "disabledBackground": "rgba(0, 0, 0, 0.12)",
                      "hover": "rgba(0, 0, 0, 0.08)",
                      "hoverOpacity": 0.08,
                      "selected": "rgba(0, 0, 0, 0.14)",
                    },
                    "augmentColor": [Function],
                    "background": Object {
                      "default": "#fafafa",
                      "paper": "#fff",
                    },
                    "common": Object {
                      "black": "#000",
                      "white": "#fff",
                    },
                    "contrastThreshold": 3,
                    "divider": "rgba(0, 0, 0, 0.12)",
                    "error": Object {
                      "contrastText": "#fff",
                      "dark": "#d32f2f",
                      "light": "#e57373",
                      "main": "#f44336",
                    },
                    "getContrastText": [Function],
                    "grey": Object {
                      "100": "#f5f5f5",
                      "200": "#eeeeee",
                      "300": "#e0e0e0",
                      "400": "#bdbdbd",
                      "50": "#fafafa",
                      "500": "#9e9e9e",
                      "600": "#757575",
                      "700": "#616161",
                      "800": "#424242",
                      "900": "#212121",
                      "A100": "#d5d5d5",
                      "A200": "#aaaaaa",
                      "A400": "#303030",
                      "A700": "#616161",
                    },
                    "primary": Object {
                      "contrastText": "#fff",
                      "dark": "#303f9f",
                      "light": "#7986cb",
                      "main": "#3f51b5",
                    },
                    "secondary": Object {
                      "contrastText": "#fff",
                      "dark": "#c51162",
                      "light": "#ff4081",
                      "main": "#f50057",
                    },
                    "text": Object {
                      "disabled": "rgba(0, 0, 0, 0.38)",
                      "hint": "rgba(0, 0, 0, 0.38)",
                      "primary": "rgba(0, 0, 0, 0.87)",
                      "secondary": "rgba(0, 0, 0, 0.54)",
                    },
                    "tonalOffset": 0.2,
                    "type": "light",
                  },
                  "props": Object {},
                  "shadows": Array [
                    "none",
                    "0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12)",
                    "0px 1px 5px 0px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 3px 1px -2px rgba(0,0,0,0.12)",
                    "0px 1px 8px 0px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 3px 3px -2px rgba(0,0,0,0.12)",
                    "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
                    "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
                    "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
                    "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
                    "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
                    "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
                    "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
                    "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
                    "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
                    "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
                    "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
                    "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
                    "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
                    "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
                    "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
                    "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
                    "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
                    "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
                    "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
                    "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
                    "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
                  ],
                  "shape": Object {
                    "borderRadius": 4,
                  },
                  "spacing": Object {
                    "unit": 8,
                  },
                  "transitions": Object {
                    "create": [Function],
                    "duration": Object {
                      "complex": 375,
                      "enteringScreen": 225,
                      "leavingScreen": 195,
                      "short": 250,
                      "shorter": 200,
                      "shortest": 150,
                      "standard": 300,
                    },
                    "easing": Object {
                      "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
                      "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
                      "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
                      "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
                    },
                    "getAutoHeightDuration": [Function],
                  },
                  "typography": Object {
                    "body1": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.875rem",
                      "fontWeight": 400,
                      "lineHeight": "1.46429em",
                    },
                    "body1Next": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1rem",
                      "fontWeight": 400,
                      "letterSpacing": "0.00938em",
                      "lineHeight": 1.5,
                    },
                    "body2": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.875rem",
                      "fontWeight": 500,
                      "lineHeight": "1.71429em",
                    },
                    "body2Next": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.875rem",
                      "fontWeight": 400,
                      "letterSpacing": "0.01071em",
                      "lineHeight": 1.5,
                    },
                    "button": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.875rem",
                      "fontWeight": 500,
                      "textTransform": "uppercase",
                    },
                    "buttonNext": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.875rem",
                      "fontWeight": 500,
                      "letterSpacing": "0.02857em",
                      "lineHeight": 1.75,
                      "textTransform": "uppercase",
                    },
                    "caption": Object {
                      "color": "rgba(0, 0, 0, 0.54)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.75rem",
                      "fontWeight": 400,
                      "lineHeight": "1.375em",
                    },
                    "captionNext": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.75rem",
                      "fontWeight": 400,
                      "letterSpacing": "0.03333em",
                      "lineHeight": 1.66,
                    },
                    "display1": Object {
                      "color": "rgba(0, 0, 0, 0.54)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "2.125rem",
                      "fontWeight": 400,
                      "lineHeight": "1.20588em",
                    },
                    "display2": Object {
                      "color": "rgba(0, 0, 0, 0.54)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "2.8125rem",
                      "fontWeight": 400,
                      "lineHeight": "1.13333em",
                      "marginLeft": "-.02em",
                    },
                    "display3": Object {
                      "color": "rgba(0, 0, 0, 0.54)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "3.5rem",
                      "fontWeight": 400,
                      "letterSpacing": "-.02em",
                      "lineHeight": "1.30357em",
                      "marginLeft": "-.02em",
                    },
                    "display4": Object {
                      "color": "rgba(0, 0, 0, 0.54)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "7rem",
                      "fontWeight": 300,
                      "letterSpacing": "-.04em",
                      "lineHeight": "1.14286em",
                      "marginLeft": "-.04em",
                    },
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": 14,
                    "fontWeightLight": 300,
                    "fontWeightMedium": 500,
                    "fontWeightRegular": 400,
                    "h1": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "6rem",
                      "fontWeight": 300,
                      "letterSpacing": "-0.01562em",
                      "lineHeight": 1,
                    },
                    "h2": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "3.75rem",
                      "fontWeight": 300,
                      "letterSpacing": "-0.00833em",
                      "lineHeight": 1,
                    },
                    "h3": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "3rem",
                      "fontWeight": 400,
                      "letterSpacing": "0em",
                      "lineHeight": 1.04,
                    },
                    "h4": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "2.125rem",
                      "fontWeight": 400,
                      "letterSpacing": "0.00735em",
                      "lineHeight": 1.17,
                    },
                    "h5": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1.5rem",
                      "fontWeight": 400,
                      "letterSpacing": "0em",
                      "lineHeight": 1.33,
                    },
                    "h6": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1.25rem",
                      "fontWeight": 500,
                      "letterSpacing": "0.0075em",
                      "lineHeight": 1.6,
                    },
                    "headline": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1.5rem",
                      "fontWeight": 400,
                      "lineHeight": "1.35417em",
                    },
                    "overline": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.75rem",
                      "fontWeight": 400,
                      "letterSpacing": "0.08333em",
                      "lineHeight": 2.66,
                      "textTransform": "uppercase",
                    },
                    "pxToRem": [Function],
                    "round": [Function],
                    "subheading": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1rem",
                      "fontWeight": 400,
                      "lineHeight": "1.5em",
                    },
                    "subtitle1": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1rem",
                      "fontWeight": 400,
                      "letterSpacing": "0.00938em",
                      "lineHeight": 1.75,
                    },
                    "subtitle2": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "0.875rem",
                      "fontWeight": 500,
                      "letterSpacing": "0.00714em",
                      "lineHeight": 1.57,
                    },
                    "title": Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                      "fontSize": "1.3125rem",
                      "fontWeight": 500,
                      "lineHeight": "1.16667em",
                    },
                    "useNextVariants": false,
                  },
                  "zIndex": Object {
                    "appBar": 1100,
                    "drawer": 1200,
                    "mobileStepper": 1000,
                    "modal": 1300,
                    "snackbar": 1400,
                    "tooltip": 1500,
                  },
                }
              }
              title=""
            >
              <RootRef
                rootRef={[Function]}
              >
                <WithStyles(Avatar)
                  aria-describedby={null}
                  className="ImageSquareAvatar-avatar-1"
                  onBlur={[Function]}
                  onFocus={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  src=""
                  title=""
                >
                  <Avatar
                    aria-describedby={null}
                    className="ImageSquareAvatar-avatar-1"
                    classes={
                      Object {
                        "colorDefault": "MuiAvatar-colorDefault-108",
                        "img": "MuiAvatar-img-109",
                        "root": "MuiAvatar-root-107",
                      }
                    }
                    component="div"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                    src=""
                    title=""
                  >
                    <div
                      aria-describedby={null}
                      className="MuiAvatar-root-107 MuiAvatar-colorDefault-108 ImageSquareAvatar-avatar-1"
                      onBlur={[Function]}
                      onFocus={[Function]}
                      onMouseLeave={[Function]}
                      onMouseOver={[Function]}
                      onTouchEnd={[Function]}
                      onTouchStart={[Function]}
                      title=""
                    />
                  </Avatar>
                </WithStyles(Avatar)>
              </RootRef>
              <Popper
                className="MuiTooltip-popper-99"
                disablePortal={false}
                id={null}
                open={false}
                placement="right"
                transition={true}
              />
            </Tooltip>
          </WithStyles(Tooltip)>
        </div>
      </Grid>
    </WithStyles(Grid)>
  </ImageSquareAvatar>
</WithStyles(ImageSquareAvatar)>
`;
